automaticRelease.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
Where is this script called from?

Reply:
Actually nowhere, that's a leftover from an unsuccessful attempt to automatically generate version identifiers. I deleted the script.